client/components/section-nav/style.scss

```diff
@@ -196,4 +196,12 @@
 // -------- Search --------
 .section-nav .search {
 	overflow: hidden;
+
+	&.is-expanded-to-container {
+		height: 45px;
+
+		@include breakpoint-deprecated( '>480px' ) {
+			height: 100%;
+		}
```

Comment:
Add specific styling for when `Search` is rendered inside the `SectionNav` component for small screens, preserving the standard for the rest of screen sizes. This avoids that the search icon is rendered in the vertical middle of the `SectionNav` component when the dropdown is open.

As such:
![image](https://user-images.githubusercontent.com/5550190/165983524-7cc79e5f-6c42-4078-8fbc-08b938b87fa0.png)